src/config/default-config.js

```diff
@@ -1,10 +1,10 @@
 const AWS = require('aws-sdk');
 const getLogger = require('../utils/getLogger');
+const { ACCOUNT_ID } = require('../api.v2/constants');
 
 const logger = getLogger();
-const { ADMIN_SUB } = require('../api.v2/constants');
 
-const githubOrganisationName = 'hms-dbmi-cellenics';
+const githubOrganisationName = process.env.AWS_ACCOUNT_ID === ACCOUNT_ID.HMS ? 'hms-dbmi-cellenics' : 'biomage-org';
```

Comment:
another potential candidate for moving to a single domain specific content

Reply:
This one is where all the variables are extracted from (including the account id, etc) so it is a little bit harder to try to move it into `getDomainSpecific`  (because `getDomainSpecific` checks this file to know which domain it is in). 

I think ideally we could try to keep the domain specific checks down to this file and `getDomainSpecific` (similarly to how in the ui we. have `next.config.js` and `getDomainSpecificContent`)